.gitignore

```diff
@@ -18,3 +18,11 @@ firebase-debug.log
 /attributes.json
 /venv/
 /local/
+
+/contrib/pyudmi/build/
```

Comment:
I think these can go in a .gitignore file in the contrib/pyudmi directory itself... makes it easier to compartmentalize